docs/hardware/devices/heltec/index.mdx

```diff
@@ -82,15 +87,19 @@ This device may have issues charging a connected battery if utilizing a USB-C to
 
 Image Source: [Heltec](<https://resource.heltec.cn/download/WiFi_LoRa32_V3/HTIT-WB32LA(F)_V3.png>)
 
-## Resources
+### Resources
 
 - Firmware file: `firmware-heltec-v3-X.X.X.xxxxxxx.bin`
-- Purchase link: [Heltec](https://heltec.org/project/wifi-lora-32-v3/)
-- Purchase link: [AliExpress](https://www.aliexpress.us/item/3256805256690400.html)
+- Purchase links
+  - [Heltec](https://heltec.org/project/wifi-lora-32-v3/)
+  - [AliExpress](https://www.aliexpress.us/item/3256805256690400.html)
 
 </TabItem>
+
 <TabItem value="Wireless Stick Lite V3">
 
+## HELTEC Lite v3
```

Comment:
IMO this would be more clear if labeled "HELTEC WSL V3"  if not "HELTEC Wireless Stick Lite V3"

We could eventually add the Wireless Stick (non lite)  as supported so for consistency it would need to be in the header.